terraform/environment/modules/environment/rds.tf

```diff
@@ -114,8 +114,8 @@ resource "aws_db_parameter_group" "postgres13-db-params" {
 
   parameter {
     name         = "log_statement"
```

Comment:
How does this change behaviour please?

Reply:
We will stop logging all queries and only log queries that take 500ms > which, according to Performance Insights, are rare